project/SparkBuild.scala

```diff
@@ -277,6 +277,7 @@ object SparkBuild extends PomBuild {
       // Google Mirror of Maven Central, placed first so that it's used instead of flaky Maven Central.
       // See https://storage-download.googleapis.com/maven-central/index.html for more info.
       "gcs-maven-central-mirror" at "https://maven-central.storage-download.googleapis.com/maven2/",
+      "jitpack" at "https://jitpack.io",
```

Comment:
Just wondering if this can cause a flakiness like the default Maven.

Due to the Apache Spark's high traffic, the Maven was flaky. So, we are using GCS Maven as the first cache here inevitably.

Reply:
Understand your concerns. We can wait for a while longer to see if `RoaringBitmap ` can resume its publish on  Maven Central